File: Task1-Server/Kolosok.Domain/Entities/Victim.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kolosok.Domain.Entities
{
    public class Victim : BaseEntity
    {
        public Guid ContactId { get; set; }
        [Required]
        public Contact Contact { get; set; }
        
        public Guid BrigadeRescuerId { get; set; }
        public BrigadeRescuer BrigadeRescuer { get; set; }
        
        public ICollection<Diagnosis> Diagnoses { get; set; }
        public ICollection<Action> Actions { get; set; }
    }
}

File: apzkr-pzpi-21-3-dzabakhidze-davyd/Task3-WebClient/src/app/models/victim.ts
import {Action} from "./action";
import {BaseEntity} from "./base-entity";
import {Contact} from "./contact";
import {BrigadeRescuer} from "./brigade-rescuer";
import {Diagnosis} from "./diagnosis";

export interface Victim extends BaseEntity {
  contactId: string;
  contact: Contact;
  brigadeRescuerId: string;
  brigadeRescuer: BrigadeRescuer;
  diagnoses: Diagnosis[];
  actions: Action[];
}
